guide/designing-products/user-research-resources.md

```diff
@@ -37,29 +37,53 @@ Below is a collection of published user research findings around Bitcoin, from a
+
+*A survey of security behaviors among three Bitcoin user archetypes, showing that cypherpunks tend to use self-managed security solutions, whereas hodlers and rookies use either custodial solutions or non-custodial wallets.*
+
 Published in February 2021 by Svetlana Abramova, Artemij Voskobojnikov, Konstantin Beznosov, and Rainer Böhme.
 
 ###### [The U in Crypto Stands for Usable: An Empirical Study of User Experience with Mobile Cryptocurrency Wallets](https://voskart.de/pdf/u_in_crypto.pdf)
+
+*An analysis of app reviews of the top five mobile cryptocurrency wallets suggesting that both new and experienced users struggle with general and domain-specific UX issues that might lead to dangerous errors and irreversible monetary losses.*
+
 Published in January 2021 by Artemij Voskobojnikov, Oliver Wiese, Masoud Mehrabi Koushki, Volker Roth, and Konstantin Beznosov.
 
 ###### [User Perception of Bitcoin Usability and Security across Novice Users](https://www.researchgate.net/publication/331286661_User_Perception_of_Bitcoin_Usability_and_Security_across_Novice_Users)
+
+*A survey of undergraduate students at Keele University concluding that Bitcoin was still in its infancy at the time of writing and required user education and a new way of thinking. The study recommends user-centered design to develop users’ mental models and deepen their understanding of digital payment technology.*
+
 Published in February 2019 by Abdulla Alshamsi & Keele University.
 
 ###### [A First Look at the Usability of Bitcoin Key Management](https://arxiv.org/pdf/1802.04351.pdf)
+
+*A broad usability evaluation of key management techniques used by six representative Bitcoin clients. Findings show that Bitcoin shares many fundamental challenges known from other domains, and also presents a unique opportunity to rethink key management for end users.*
+
 Published in February 2018 by Shayan Eskandari, David Barrera, Elizabeth Stobert, and Jeremy Clark.
 
 ###### [An Evaluation of User Attitudes Towards Anonymity in Bitcoin](https://www.diva-portal.org/smash/get/diva2:1141672/FULLTEXT01.pdf)
+
+*A survey of 50 users on the [r/bitcoin subreddit](https://www.reddit.com/r/Bitcoin/) about their attitudes towards anonymity in the Bitcoin network.*
+
 Published in the spring of 2017 by Mihkel Pajunen.
 
 ###### [The Other Side of the Coin: User Experiences with Bitcoin Security and Privacy](https://publications.sba-research.org/publications/TheOtherSideOfTheCoin_FC16preConf.pdf)
+
+*A survey and follow-up interviews of Bitcoin users to determine bitcoin management strategies and identify how they deploy security measures to protect their keys and bitcoins. It was found that many do not use all security capabilities of their selected applications and have significant misconceptions on how to remain anonymous and protect their privacy.*
+
 Published in May 2017 by Katharina Krombholz, Aljosha Judmayer, Matthias Gusenbauer, and Edgar Weippl.
 
 ## Surveys
```

Comment:
How is the distinction between surveys and research papers being made? Most of the research papers are surveys and some surveys could be classed as research papers.

Reply:
When I first gathered the resources, I didn't read through everything and created this loose grouping. But reading through them during this PR work, I also realized that things need to be re-ordered. I decided to keep that for a separate PR though.